apps/studio/src/lib/db/clients/sqlite.ts

```diff
@@ -135,7 +135,7 @@ export class SqliteClient extends BasicDatabaseClient<SqliteResult> {
 
   async listTableColumns(db: string, table?: string, _schema?: string): Promise<ExtendedTableColumn[]> {
     if (table) {
-      const sql = `PRAGMA table_info(${SD.escapeString(table, true)})`;
+      const sql = `PRAGMA table_xinfo(${SD.escapeString(table, true)})`;
```

Comment:
One big change we'll need to go along with this:

When someone is editing table data, Beekeeper would let them edit this column. That's wrong as you can't update a generated column.

## This PR will need:
- Some way for the app to understand that a column is read only and disable editing of data in that column. 
- Some visual indicator in the tabletable column header to indicate that the column is generated (maybe edit_off from materialize)
- Some sort of addition to the table structure view so that folks can see it is generated. We don't need support for creating generated_columns as part of this PR
- Additions to the `TableColumn` interface to indicate the column is read only. Maybe something like `readOnly`, or `generated`.

Reply:
Right, totally missed that.

Reply:
Many databases we support can have generated columns. So I'm probably gonna add this for those databases as well, as long as it's not too complicated.